mac/xdag.xcodeproj/project.pbxproj

```diff
@@ -667,7 +667,7 @@
 				CLANG_ENABLE_OBJC_ARC = YES;
 				CODE_SIGN_STYLE = Automatic;
 				GCC_C_LANGUAGE_STANDARD = gnu11;
-				GCC_OPTIMIZATION_LEVEL = 3;
```

Comment:
You changed the optimization level. Is it correct?

Reply:
yes, it for xcode debug version.